.eslintrc.js

```diff
@@ -76,5 +76,11 @@ module.exports = {
         '@typescript-eslint/no-var-requires': 'off',
       },
     },
+    {
+      files: 'packages/big-design-icons/**/*.tsx',
+      rules: {
+        'react/jsx-sort-props': 1,
```

Comment:
I'll use `"warn"` then since that's what we have in [`@bigcommerce/eslint-config`](https://github.com/bigcommerce/eslint-config/blob/0b1c5a36c76b8679610ac10dc3c9a3debbd78c1e/packages/eslint-config/configs/react.js#L37)

Reply:
lol, yeah 😅